index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Youssef's Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="header" id="container">
+      <a href="#" class="logo">Youssef</a>
+      <nav class="navbar">
+        <ul>
+          <li><a href="#container">Home</a></li>
+          <li><a href="#about">About me</a></li>
+          <li><a href="#projects">Projects</a></li>
+          <li><a href="#contact">Contact me</a></li>
+        </ul>
+      </nav>
+    </div>
+
+    <!-- main---------------------- -->
+
+    <div class="main" id="container">
+      <p class="developer">CSS and HTML develober</p>
+      <h1 class="description1">
+        Hi, I am <span>Youssef</span> <br />and I am passionate about what I am
+        doing
+      </h1>
+    </div>
+
+    <!-- about---------------------- -->
+
+    <div class="about" id="about">
+      <h1 class="developer">About me..</h1>
+      <p class="description2">bla bla bla</p>
+      <div>
+        <p class="skills" onclick="opentab('skills')">My skills</p>
+      </div>
+      <div>
+        <p class="description3">Bla bla Bla bla</p>
+      </div>
+    </div>
+
+    <!-- projects-------------------- -->
+
+    <div class="projects" id="projects">
+      <p class="description3">My projets..</p>
```

Comment:
Same here, for now, you can add just one-two images so you use display flex for example :)